pytorch/README.md

```diff
@@ -208,6 +208,99 @@ To add these files correctly please follow the steps described below.
 
 ---
 
+The image below is an extension of the Multinode container designed to run Huggingface Generative AI scripts
```

Comment:
Do we need to duplicate this BKM in the README? Or is there some new instructions we can provide to customers?

Reply:
I wouldn't duplicate it. Maybe just say that the container has the typical installations needed to run and fine tune PyTorch generative text models from Hugging Face. You can also mention that it can be used to run multinode jobs using the same instructions from the IPEX multi-node container.

If you want to show a single node example command, you could have going a volume mount of `../workflows/charts/huggingface-llm/scripts:/workspace/scripts` and executing `python /workspace/scripts/finetune.py ...`

Reply:
Modified README in this [commit](https://github.com/intel/ai-containers/pull/146/commits/6d781654ec53ebbdede0130abf37c8a23adace6c)